polyfills/Symbol/config.json

```diff
@@ -8,7 +8,7 @@
 		"safari": "<9",
 		"chrome": "< 50",
 		"firefox": "< 40",
-		"ios_saf": "<9",
+		"ios_saf": "<10",
```

Comment:
Yep, but I don't understand why we were serving it to 10 before. Wonder if this was just people using the `always` flag